sisl/geom/flat.py

```diff
@@ -70,3 +70,124 @@ def graphene(bond=1.42, atoms=None, orthogonal=False):
     if atoms is None:
         return honeycomb(bond, Atom(Z=6, R=bond * 1.01), orthogonal)
     return honeycomb(bond, atoms, orthogonal)
+
+
+def _finish_slab(g, size, vacuum):
+    g = g.repeat(size[1], 1).repeat(size[0], 0)
+    if vacuum is not None:
+        g.cell[2, 2] += vacuum
+        g.set_nsc([3, 3, 1])
+    else:
+        g.set_nsc([3, 3, 3])
+    if np.all(g.maxR(True) > 0.):
+        g.optimize_nsc()
+    return g
+
+
+def fcc100(alat, atoms, size=(1, 1, 2), vacuum=None):
+    """ Face-centered cubic (100) surface
+
+    Parameters
+    ----------
+    alat : float
+        lattice constant of the fcc crystal
+    atoms : Atom
+        the atom that the crystal consists of
+    size : 3-array, optional
+        slab size along the lattice vectors
+    vacuum : float, optional
+        distance added to the third lattice vector to separate
+        the slab from its periodic images
+
+    See Also
+    --------
+    fcc110 : construction of fcc(110) surface
+    fcc111 : construction of fcc(111) surface
+    """
+    sc = SuperCell(np.array([np.sqrt(0.5), np.sqrt(0.5), 0.5]) * alat)
```

Comment:
here I would prefer
`np.array([0.5 ** 0.5, 0.5 ** 0.5, 0.5]) * alat` for just a bit clearer reading.

Reply:
OK